src/frontend/planner_test/tests/testdata/over_window_function.yaml

```diff
@@ -352,6 +352,33 @@
   stream_error: |-
     Feature is not yet implemented: General version of streaming over window is not implemented yet
     Tracking issue: https://github.com/risingwavelabs/risingwave/issues/9124
+- name: split diff order_by and partition_by in window
+  sql: |
+    create table t(x int, y int, z int);
+    select x, y, rank0, rank1, rank2, rank3 from
+      (select *, row_number() over(PARTITION BY x ORDER BY y) rank0, row_number() over(PARTITION BY z ORDER BY x) rank1, rank() over(PARTITION BY x ORDER BY y) rank2, rank() over(PARTITION BY y ORDER BY x) rank3 from t)
+    where rank0 = 1
```

Comment:
I think it can be removed and we should not mix them in the test :thinking:

Reply:
and maybe we can add more kinds of over-window functions that can contain input refs of the input, such as `sum(x)`